.github/workflows/eslint-config.yml

```
@@ -46,4 +46,4 @@ jobs:
       - run: yarn publish
         working-directory: ${{ env.PROJECT_DIR }}
         env:
-          NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
+          NODE_AUTH_TOKEN: ${{ secrets.NPM_API_KEY }}
```

I can make the old version work.

```suggestion
          NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
```

---

Both work now. When this merges and I know which one you actually want to use, I'll delete the other.

---

Ok, now only NPM_TOKEN works.